packages/web3torrent/src/pages/welcome/Welcome.scss

```diff
@@ -31,9 +34,58 @@ section.fill {
   }
 }
 
-.subtitle {
+div.subtitle {
   font-size: 18px;
   line-height: 32px;
   text-align: center;
-  margin: 43px auto 20px;
+  margin: 20px auto 10px auto;
+
+  p {
+    margin: 0;
+  }
+  .actions-container {
+    margin-bottom: 0;
+    button {
+      margin-top: 0px;
+      margin-bottom: 0px;
+      margin-left: 5px;
+      margin-right: 5px;
+    }
+  }
+}
+
+.pulse {
+  display: block;
+  cursor: pointer;
+  box-shadow: $c-orange;
+  animation: pulse 1.5s infinite;
```

Comment:
![Jun-18-2020 14-20-38](https://user-images.githubusercontent.com/1833419/85025222-031deb00-b16f-11ea-876d-2b952a894993.gif)

Reply:
I personally find this slightly off-putting. It feels a bit like the button is winking at me in a seductive manner. I know I'm meant to push it, but I'm wondering if it would be wrong to do so.

What would it look like if we just used the thicker border?

Reply:
But not too thick...

Reply:
It looks like this:
![Screenshot 2020-06-18 at 15 57 59](https://user-images.githubusercontent.com/1833419/85036838-85f97280-b17c-11ea-9062-f9622c82e401.png)

Reply:
What about making it pulse twice?

Reply:
https://github.com/statechannels/monorepo/pull/2233